stewards.json

```diff
@@ -1,6 +1,6 @@
 [
     {
-        "ca": "standard_crypto/server_ca.crt",
-        "endpoint": "https://steward.sommelier.standardcryptovc.com:5734"
+        "ca": "forbole/server_ca.crt",
+        "endpoint": "https://steward.sommelier.forbole.com:5734"
```

Comment:
You need to append an entry to this list rather than overwriting it. You'll need to merge main and update.

Reply:
Done